packages/e2e.sandbox.todoapp/src/main.tsx

```diff
@@ -3,11 +3,11 @@ import ReactDOM from "react-dom/client";
 import App from "./App.js";
 import "./index.css";
 import { OsdkProvider2 } from "@osdk/react/experimental";
-import { $, store } from "./foundryClient.js";
+import { $ } from "./foundryClient.js";
 
 ReactDOM.createRoot(document.getElementById("root")!).render(
   <React.StrictMode>
-    <OsdkProvider2 client={$} store={store}>
+    <OsdkProvider2 client={$}>
```

Comment:
I'll optionally create one for you now if you dont give one to me. Cleans up docs quite a bit